examples/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+project(mococrw-examples)
+
+if(BUILD_EXAMPLES)
+    message(STATUS "Compiling examples")
+    set(EXAMPLE_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+    FIND_PACKAGE(Boost COMPONENTS program_options REQUIRED)
```

```suggestion
    find_package(Boost COMPONENTS program_options REQUIRED)
```

Lower case, please, hardly anybody still writes CMake commands in upper case.